CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - Minor: Received Twitch messages now use the exact same timestamp (obtained from Twitch's server) for every Chatterino user instead of assuming message timestamp on client's side. (#3021)
 - Minor: Received IRC messages use `time` message tag for timestamp if it's available. (#3021)
 - Bugfix: Fixed "smiley" emotes being unable to be "Tabbed" with autocompletion, introduced in v2.3.3. (#3010)
+- Bugfix: PubSub didn't properly try to resolve pending listens when the pending listens list was larger than 50. (#3037)
```

Comment:
```suggestion
- Bugfix: Fixed PubSub not properly trying to resolve pending listens when the pending listens list was larger than 50. (#3037)
```
The original variation makes sense, but it reads as "Bugfix: This was broken" and doesn't sound like this PR specifically fixed/fixes it.